spec/factories/tv_series_viewings.rb

```diff
@@ -1,7 +1,7 @@
 FactoryBot.define do
   factory :tv_series_viewing do
     user
-    title { FFaker::HipsterIpsum.phrase }
+    sequence(:title) { |n| "TV Series #{n}" }
```

Comment:
The ipsum generated here has some icky symbols. I went with a simple sequence here instead.